HousingFinanceInterimApi/V1/Domain/ActionDiaryAuxDomain.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using HousingFinanceInterimApi.JsonConverters;
+using Newtonsoft.Json;
+
+namespace HousingFinanceInterimApi.V1.Domain
+{
+    public class ActionDiaryAuxDomain
+    {
+        public long Id { get; set; }
+
+        [JsonProperty("Tenancy Agreement Ref")]
+        public string TenancyAgreementRef { get; set; }
+
+        [JsonProperty("Rent Account")]
+        public string RentAccount { get; set; }
+
+        [JsonProperty("Action Code (Don't need fill)")]
```

Comment:
What does Don't need fill mean here?

Reply:
It's a spreadsheet column name. The users don't need to fill this column because it's auto-completed based on action drop-down selection but the system uses the code. 

![image](https://user-images.githubusercontent.com/78608163/134336082-90c11dc0-e1a1-4956-bc5e-bef9cc11c016.png)